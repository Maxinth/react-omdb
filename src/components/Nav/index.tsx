import { NavBox, LogoBox, Span } from "./nav.styled";
import React from "react";

const Nav = () => {
  return (
    <NavBox>
      <LogoBox>
        <Span>MyTestApp</Span>
      </LogoBox>
    </NavBox>
  );
};

export default Nav;
