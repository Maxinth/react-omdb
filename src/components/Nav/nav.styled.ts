import styled from "styled-components";

const LogoBox = styled.div``;
const Span = styled.div``;

const NavBox = styled.nav`
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 16px 106px 17.42px 107px;
  background-color: #292929;
  transition: all 0.2s;
  ${LogoBox} {
    padding: 7.83px 9.68px 7.31px 10.63px;
    :border-radius: 0px;
    color: #ffffff;
    width: 108px;
    height: 33.58px;
    border: 1px solid #fff;
    ${Span} {
      display: flex;
      justify-content: center;
      align-items: center;
      width: 87.68px;
      height: 18.43px;
      padding: 7.83px 9.68px 7.31px 10.63px;
    }
  }
  @media (min-width: 1440px) {
    justify-content: flex-start;
    padding: 40px 1170px 40px 77px;
  }
`;

export { NavBox, LogoBox, Span };
