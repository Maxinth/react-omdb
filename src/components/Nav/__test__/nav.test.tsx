import Nav from "..";
import renderer from "react-test-renderer";
import React from "react";

it("renders a snapshot of <Nav />", () => {
  const nav = renderer.create(<Nav />).toJSON();
  expect(nav).toMatchSnapshot();
});
