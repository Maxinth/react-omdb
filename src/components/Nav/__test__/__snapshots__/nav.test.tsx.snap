// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <Nav /> 1`] = `
<nav
  className="sc-dkPtRN eFACGC"
>
  <div
    className="sc-bdvvtL"
  >
    <div
      className="sc-gsDKAQ"
    >
      MyTestApp
    </div>
  </div>
</nav>
`;
